cmd/yes/yes.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+)
+
+func main() {
+	args := os.Args[1:]
+	for range 10 {
+		if len(args) > 0 {
```

Comment:
why are you looping for only 10 times? try to run yes and check its behavior, it doesn't loop only 10 times, doesn't print yes if there's no string input, nor it sleeps for a second.
also, range shouldn't be used with int value